### PR TITLE
Wizard routing refactor

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
## Description

This PR refactors user account routing to use routing components from react-router. Included is some cleanup of retrieving the active page on a wizard.

## PR Checklist
- [ ] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

File: lib/components/user/user-account-screen.tsx [hunk 1]
```diff
@@ -193,45 +194,30 @@ class UserAccountScreen extends Component<Props> {
...
+                        onCreate={this._handleCreateNewUser}
+                      />
+                    </Route>
+                    <Route path={MOBILITY_PATH}>
+                      <MobilityWizard formikProps={newFormikProps} />
+                    </Route>
+                    <Route>
+                      <ExistingAccountDisplay {...newFormikProps} />
+                    </Route>
+                  </Switch>
```
Comment (miles-grant-ibigroup) [inline lib/components/user/user-account-screen.tsx:219]: I think this change is causing the entire page to refresh when clicking some links in the account screen, but it's a worthy trade off in exchange for the benefits this PR provides!

**Review by miles-grant-ibigroup — APPROVED** [review]
Small behavior change but I think it's acceptable. Code is clean

**Review by josh-willis-arcadis — APPROVED** [review]
Met with Binh to discuss the changes and the PR process.

**Review by amy-corson-ibigroup — APPROVED** [review]
Looks good to me! Thank you!

**Pull request merged**